config/config.yaml

```diff
@@ -26,4 +26,9 @@ qbox/kodo: # github repository name, which will override the settings in qbox or
     workDir: "src/qiniu.com/kodo"
     command: staticcheck
     args: ["-debug.no-compile-errors=true", "./..."]
-    run_if_changed: ["src/qiniu.com/kodo/.*\\.go$"] # only run if changed files match the regex
\ No newline at end of file
+    run_if_changed: ["src/qiniu.com/kodo/.*\\.go$"] # only run if changed files match the regex
+
+qbox/3rd-ffmpeg: # github repository name, which will override the settings in qbox org
```

Comment:
why do this config? since no custom configs

Reply:
https://github.com/qbox/3rd-ffmpeg